plugins/package-managers/pub/src/main/kotlin/model/Lockfile.kt

```diff
@@ -41,8 +39,6 @@ import kotlinx.serialization.encoding.Decoder
 
 import org.ossreviewtoolkit.plugins.packagemanagers.pub.model.PackageInfo.Description
 
-private val YAML = Yaml(configuration = YamlConfiguration(strictMode = false))
```

Comment:
Hm, personally sacrificing encapsulation for these few bytes of memory is not worth it.
What do you think about the middle ground of keeping it private, but turning it into a function?

Reply:
Why would encapsulation even be required here? I mean, these two files *are* supposed to be deserialized in exactly the same way, so it makes sense and is correct to share the same `Yaml` instance, or?

Background: I was originally doing this in the context of adding more tests for deserialization (which turned out to be unnecessary). As part of that, I needed access to `YAML`, but I could make either instance `internal` as it would then have conflicted with the other `private` one.